examples/graph/src/main.rs

```diff
@@ -1,14 +1,14 @@
 #![allow(non_snake_case)]
 #![allow(unused_variables)]
-#[macro_use]
-extern crate qmetaobject;
-use qmetaobject::scenegraph::*;
-use qmetaobject::*;
+
 #[macro_use]
 extern crate cstr;
```

Comment:
yeah, all these extern crate could also be cleaned up, but that's unrelated.

Reply:
I… kinda missed out when cpp and cstr became 2018 edition-compatible. Either that or qmetaobject-rs was at 2015. Anyway, I remember clear as day, it didn't work without `#[macro_use] extern crate`.

Working on a patch...